packages/client/src/forms/user/fieldDefinitions/createUser.ts

```diff
@@ -13,251 +13,252 @@ import {
+            validator: [{ operation: 'englishOnlyNameFormat' }],
+            mapping: {
+              mutation: {
+                operation: 'fieldToNameTransformer',
+                parameters: ['en', 'firstNames']
+              },
+              query: {
+                operation: 'nameToFieldTransformer',
+                parameters: ['en', 'firstNames']
+              }
             }
-          }
-        },
-        {
-          name: 'username',
-          type: TEXT,
-          label: userFormMessages.username,
-          previewGroup: 'userNameGroup',
-          required: false,
-          initialValue: '',
-          validator: [],
-          readonly: true,
-          hidden: true
-        },
-        {
-          name: 'phoneNumber',
-          type: TEXT,
-          label: userFormMessages.phoneNumber,
-          required: window.config.USER_NOTIFICATION_DELIVERY_METHOD === 'sms',
-          initialValue: '',
-          validator: [{ operation: 'phoneNumberFormat' }],
-          mapping: {
-            mutation: {
-              operation: 'msisdnTransformer',
-              parameters: ['user.mobile']
-            },
-            query: {
-              operation: 'localPhoneTransformer',
-              parameters: ['user.mobile']
+          },
+          {
+            name: 'familyNameEng',
+            type: TEXT,
+            label: userFormMessages.lastNameEn,
+            required: true,
+            initialValue: '',
+            validator: [{ operation: 'englishOnlyNameFormat' }],
+            mapping: {
+              mutation: {
+                operation: 'fieldToNameTransformer',
+                parameters: ['en', 'familyName']
+              },
+              query: {
+                operation: 'nameToFieldTransformer',
+                parameters: ['en', 'familyName']
+              }
             }
-          }
-        },
-        {
-          name: 'email',
-          type: TEXT,
-          label: userFormMessages.email,
-          required: window.config.USER_NOTIFICATION_DELIVERY_METHOD === 'email',
-          initialValue: '',
-          validator: [{ operation: 'emailAddressFormat' }]
-        },
-        {
-          name: 'nid',
-          type: TEXT,
-          label: userFormMessages.NID,
-          required: false,
-          initialValue: '',
-          validator: [
-            {
-              operation: 'validIDNumber',
-              parameters: [NATIONAL_ID]
+          },
+          {
+            name: 'username',
+            type: TEXT,
+            label: userFormMessages.username,
+            previewGroup: 'userNameGroup',
+            required: false,
+            initialValue: '',
+            validator: [],
+            readonly: true,
+            hidden: true
+          },
+          {
+            name: 'phoneNumber',
+            type: TEXT,
+            label: userFormMessages.phoneNumber,
+            required: window.config.USER_NOTIFICATION_DELIVERY_METHOD === 'sms',
```

Comment:
Essentially this change from Rabiul wasnt working.  window.config hasnt been updated at this point